libparsec/crates/client/src/workspace/store/mod.rs

```diff
@@ -191,9 +192,14 @@ impl WorkspaceStore {
             certificates_ops,
             current_view_cache: Mutex::new(CurrentViewCache::new(Arc::new(root_manifest.into()))),
             storage: AsyncMutex::new(Some(storage)),
+            prevent_sync_pattern: pattern.clone(),
         })
     }
 
+    fn prevent_sync_pattern(&self) -> &Regex {
```

Comment:
You can remove the getter and go directly with `self.prevent_sync_pattern` since this is private stuff

Reply:
done